server/rating.js

```diff
@@ -4,46 +4,57 @@ import fetch from "node-fetch";
 export const displayRating = express.Router();
 
 async function getRatingData(id) {
-  const response = await fetch(
-    "https://plankton-app-xhkom.ondigitalocean.app/api/reviews?filters%5Bmovie%5D=" +
-      id
-  );
-  const reviewData = await response.json();
-  const data = reviewData.data;
-  return data;
+    const response = await fetch(
+        "https://plankton-app-xhkom.ondigitalocean.app/api/reviews?filters%5Bmovie%5D=" +
+        id
+    );
+    const reviewData = await response.json();
+    const data = reviewData.data;
+    return data;
 }
 
 export function roundRating(passInData) {
-  //removing value of null if it exist
-  let nullValue = null;
-  let filterData = passInData.filter((item) => item !== nullValue);
+    //removing value of null if it exist
+    let nullValue = null;
+    let filterData = passInData.filter((item) => item !== nullValue);
 
-  let sum = filterData.reduce(function (a, b) {
-    return a + b;
-  });
+    let sum = filterData.reduce(function(a, b) {
+        return a + b;
+    });
 
-  let result = sum / passInData.length;
+    let result = sum / passInData.length;
 
-  let roundedRating = Math.round(result * 10) / 10;
+    let roundedRating = Math.round(result * 10) / 10;
 
-  return roundedRating;
+    return roundedRating;
+}
+const API_BASE = "https://plankton-app-xhkom.ondigitalocean.app/api";
+export async function loadOMDB(id) {
+    const rev = await fetch(API_BASE + "/movies/" + id);
+    const payload = await rev.json();
+
+    const IMDB_ID = payload.data.attributes.imdbId;
+    const OMDB_API = await fetch(
+        `http://www.omdbapi.com/?i=${IMDB_ID}&apikey=951b6bb2`
```

Comment:
When using APIs that require API keys, it's always a good idea to make sure API keys are not included in the code, but stored elsewhere. A common place to store them is in environment variables.

The problem with having them in the code is that anyone with access to the code can now steal your API credentials. Since the source code for this project is publicly available, that means everyeone.